tests/test_handlers.py

```diff
@@ -135,7 +139,7 @@ def check_empty_request(self, log, handler):
 
 @override_settings(AXES_HANDLER="axes.handlers.database.AxesDatabaseHandler")
 class ResetAttemptsTestCase(AxesHandlerBaseTestCase):
-    """ Resetting attempts is currently implemented only for database handler """
+    """Resetting attempts is currently implemented only for database handler"""
```

Comment:
Whitespace and stylistic changes should ideally be separated into a separate commit and PR.